src/rust/engine/Cargo.toml

```diff
@@ -9,10 +9,15 @@ publish = false
 # tokio_unstable is necessary for access to the `id`-related functions of tokio::task::JoinSet.
 # See https://docs.rs/tokio/1.21.1/tokio/task/struct.JoinSet.html#method.join_next_with_id.
 
+[profile.dev]
+# Increase the optimization level of the dev profile slightly, as otherwise `rule_graph`
+# solving takes prohibitively long.
+opt-level = 1
+
 [profile.release]
-# Enable debug symbols in the `release` profile: doesn't add a noticeable overhead in size/speed,
```

Comment:
Did you happen to measure the size decrease of this change?

Reply:
Looks like about 42% smaller for Linux wheels, and no change for macOS wheels: https://gist.github.com/stuhood/830d49ddf63276f63a0f9d76090f9d45

Reply:
Big win, nice!